copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,857 @@
+{
```

Comment:
<p>Sorry, maybe one or two more things - for validation, since we're already reading the file, we could check for <code>copc: true</code> as validation rather than just file creation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>